themes/powerbash10k/powerbash10k.theme.sh

```diff
@@ -0,0 +1,367 @@
+#! bash oh-my-bash.module
+
+# powerbash10k theme for oh-my-bash, inspired by powerlevel10k zsh theme
+# made by awerebea, based on brainy theme by MunifTanjim
+
+# Helpers
+function __pb10k_remove_empty_elements {
+    IFS="|" read -ra origin_array <<< "$1"
```

Comment:
I'm not sure if it is intensional or not, but why doesn't this function have declarations of (seemingly) local variables despite that all the other functions seem to carefully declare local variables? Are these variables used outside this function? Maybe, can we make these variables local?

```suggestion
    local origin_array new_array element trimmed
    IFS="|" read -ra origin_array <<< "$1"
```